frontend/src/pages/projects/screens/detail/connections/ConnectionsList.tsx

```diff
@@ -71,17 +73,25 @@ const ConnectionsList: React.FC = () => {
             iconImage={typedEmptyImage(ProjectObjectType.connections)}
             imageAlt="create a connection"
             createButton={
-              <Button
-                key={`action-${ProjectSectionID.CONNECTIONS}`}
-                variant="primary"
-                data-testid="create-connection-button"
-                onClick={() => {
-                  setManageConnectionModal({});
-                }}
-                isDisabled={enabledConnectionTypes.length === 0}
-              >
-                Create connection
-              </Button>
+              <>
+                <Button
+                  variant="primary"
+                  data-testid="create-connection-button"
+                  aria-describedby={
+                    enabledConnectionTypes.length === 0 ? 'no-connection-types-tooltip' : undefined
+                  }
+                  isAriaDisabled={enabledConnectionTypes.length === 0}
+                  onClick={() => {
+                    setManageConnectionModal({});
+                  }}
+                  ref={tooltipRef}
+                >
+                  Create connection
+                </Button>
+                {enabledConnectionTypes.length === 0 && (
+                  <Tooltip content="No connection types available" triggerRef={tooltipRef} />
```

Comment:
Missing ID to go with the `aria-describedby` attribute.
```suggestion
                  <Tooltip id="no-connection-types-tooltip" content="No connection types available" triggerRef={tooltipRef} />
```

Also noticed that the `Add connection` button doesn't have a tooltip when there is an existing connection but no connection types.

![image](https://github.com/user-attachments/assets/db6d1b22-80c8-4aa6-b045-3ac11c692af0)

Reply:
Thank you @christianvogt - applied these fixes